Source/Fuse.Controls/Tests/Fuse.Controls.Test.Helpers/PanelTester.uno

```diff
@@ -122,6 +125,8 @@ namespace Fuse.Controls.Test
 				LayoutTestHelper.TestElementLayout(root, panelToTest, int2(800,600),
 					root.SnapToPixelsSize(float2(100, 71)), float2(800 - 100 - root.SnapToPixelsPos(21.4f),
 					root.SnapToPixelsPos(5.3f)));
+
+				root.Children.Remove(parent);
```

Comment:
It is now illegal to be a child of more than more parent. This test violated that.